DfE.FindInformationAcademiesTrusts/Services/Export/ExportService.cs

```diff
@@ -20,494 +17,51 @@ public class ExportService(
-        SetTextCell(worksheet, rowNumber, 2, academy.Urn);
-        SetTextCell(worksheet, rowNumber, 3, academy.LocalAuthority ?? string.Empty);
-        SetTextCell(worksheet, rowNumber, 4, academy.TypeOfEstablishment ?? string.Empty);
-        SetTextCell(worksheet, rowNumber, 5, academy.UrbanRural ?? string.Empty);
-
-        SetDateCell(worksheet, rowNumber, 6, ofstedData.DateAcademyJoinedTrust);
-        
-        SetTextCell(worksheet, rowNumber, 7, currentRating.OverallEffectiveness.ToDisplayString(true));
-        SetTextCell(worksheet, rowNumber, 8,
-            IsOfstedRatingBeforeOrAfterJoining(
-                currentRating.OverallEffectiveness,
-                ofstedData.DateAcademyJoinedTrust,
-                currentRating.InspectionDate
-            )
-        );
-        SetDateCell(worksheet, rowNumber, 9, currentRating.InspectionDate);
-
-        SetTextCell(worksheet, rowNumber, 10, previousRating.OverallEffectiveness.ToDisplayString(false));
-        SetTextCell(worksheet, rowNumber, 11,
-            IsOfstedRatingBeforeOrAfterJoining(
-                previousRating.OverallEffectiveness,
-                ofstedData.DateAcademyJoinedTrust,
-                previousRating.InspectionDate
-            )
-        );
-        SetDateCell(worksheet, rowNumber, 12, previousRating.InspectionDate);
-        
-        SetTextCell(worksheet, rowNumber, 13, pupilNumbersData.PhaseOfEducation ?? string.Empty);
-
-        SetTextCell(worksheet, rowNumber, 14,
-            $"{pupilNumbersData.AgeRange.Minimum} - {pupilNumbersData.AgeRange.Maximum}"
-        );
-
-        SetTextCell(worksheet, rowNumber, 15, pupilNumbersData.NumberOfPupils?.ToString() ?? string.Empty);
-        SetTextCell(worksheet, rowNumber, 16, pupilNumbersData.SchoolCapacity?.ToString() ?? string.Empty);
-        SetTextCell(worksheet, rowNumber, 17, percentageFull > 0 ? $"{percentageFull}%" : string.Empty);
-        SetTextCell(
-            worksheet,
-            rowNumber,
-            18,
-            freeSchoolMealsData.PercentageFreeSchoolMeals.HasValue
-                ? $"{freeSchoolMealsData.PercentageFreeSchoolMeals}%"
-                : string.Empty
-        );
-
-        SetTextCell(worksheet, rowNumber, 19,
-            freeSchoolMealsData.LaAveragePercentageFreeSchoolMeals > 0
-                ? $"{Math.Round(freeSchoolMealsData.LaAveragePercentageFreeSchoolMeals, 1)}%"
-                : string.Empty
-        );
-        SetTextCell(worksheet, rowNumber, 20,
-            freeSchoolMealsData.NationalAveragePercentageFreeSchoolMeals > 0
-                ? $"{Math.Round(freeSchoolMealsData.NationalAveragePercentageFreeSchoolMeals, 1)}%"
-                : string.Empty
-        );
+        return new AcademiesBuilder()
+            .WriteTrustInformation(trustSummary)
+            .WriteHeaders()
+            .WriteRows(academiesDetails, academiesOfstedRatings, academiesPupilNumbers, academiesFreeSchoolMeals)
+            .Build();
```

Comment:
We should move each of these methods into an `Export` method in each of the relevant builders then DI the appropriate builder into the place currently calling the export service. This will encapsulate the responsibility for knowing how to build the spreadsheet completely within the builder